whois-api/src/test/java/net/ripe/db/whois/api/fulltextsearch/FullTextAnalyzerTest.java

```diff
@@ -0,0 +1,58 @@
+package net.ripe.db.whois.api.fulltextsearch;
+
+import com.google.common.collect.Lists;
+import net.ripe.db.whois.api.fulltextsearch.FullTextAnalyzer.Operation;
+import org.apache.lucene.analysis.TokenStream;
+import org.apache.lucene.analysis.tokenattributes.CharTermAttribute;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.in;
+import static org.hamcrest.Matchers.is;
+
+/**
+ * The {@link org.apache.lucene.analysis.Analyzer} used in whois does *not* split on punctuation at the moment.
+ * This has been done to support finding objects based on exact key containing interpunction.
+ * Downside of not stripping the punctuation is that the index is blown up considerably because for example we store both
+ * "members" and "members:" in the index.
+ * We probably want to implement a filter that strips most punctuation but also emits punctuation that is part of an object's key.
+ * This test has been added to more quickly find the cause of changes in fulltext search results as a result of Lucene upgrades or Analyzer changes :)
+ */
+public class FullTextAnalyzerTest {
```

Comment:
This test class is not related to the Lucene 8 update? Remove from this PR?

Reply:
It is something I have added as a result of losing time figuring out why the behaviour was different after a Lucene major version bump, trying to use the StandardAnalyzer instead but then figuring out that we can't and I want to have it documented and under test for the next time we do a major version upgrade and run into the same problem. So IMHO yes it is related to the Lucene 8 upgrade.

Reply:
OK! Thanks. I confused this with Jacopo's changes for the logsearch analyser, but they are not related.